browser/bundles/o-errors.js

```diff
@@ -1,3 +1,11 @@
+// for safety polyfill window.console
+window.console = window.console || {
+	log: function () { },
+	info: function () { },
+	warn: function () { },
+	error: function () { },
```

Comment:
Super minor: trailing comma (only mentioning it as there are similar corrections elsewhere in the PR)